agent/util-scripts/test-bin/rpmdev-vercmp

```diff
@@ -3,18 +3,27 @@
 args="${@}"
 echo "${0} ${args}" >> ${_testlog}
 
+# We are mocking the behavior of rpmdev-vercmp where the behavior of this
+# mock is directed by the 2nd arguments name.  This allows specific test
+# cases to request a certain behavior from rpmdev-vercmp.
```

Comment:
Given how much trouble I'm having deciding whether the code below is correct (as evidenced by the comments that I've made...), I'm thinking that this comment here needs significant expansion.